internal/exec/stages/files/units.go

```diff
@@ -151,7 +152,16 @@ func (s *stage) createSystemdPresetFile(presets map[string]*Preset) error {
 		return err
 	}
 	hasInstanceUnit := false
-	for _, value := range presets {
+
+	// sort the units before writing to the systemd presets file
```

Comment:
It's not completely obvious why we should do the sorting, so let's make sure to explain here.

Reply:
I updated the comment to be more descriptive.